certification/internal/engine/engine.go

```diff
@@ -39,7 +39,9 @@ import (
 // CraneEngine implements a certification.CheckEngine, and leverage crane to interact with
 // the container registry and target image.
 type CraneEngine struct {
-	Config certification.Config
+	// DockerConfig is the credential required to pull the image.
+	DockerConfig string
+	// Config certification.Config
```

Comment:
[Note To Reviewers]: This is an example of a place where we passed the full-sized configuration vs. specific components we needed previously, and this PR changes that.

Reply:
should like 44 be removed? it is commented out correct?

Reply:
Yep, I'll remove it. Good catch